src/selectors/metadata.js

```diff
@@ -94,6 +94,20 @@ const definitionsModule = memoize(moduleName =>
 	createSelector(definitions, definitions => definitions.get(moduleName)),
 );
 
+export const mappedDefinitionsSelector = memoize(moduleName =>
+	createSelector(
+		definitionsModule(moduleName),
+		currentLocaleOrDefault,
+		(definitions, locale, key, defaultValue = key) => {
+			return definitions.map(definition => {
+				return definition
+					.set("type", definition.get("isSharedEntity") === true ? definitionType.shared : definitionType.embedded)
+					.set("name", getLocalization(definition?.get("displayName"), locale, defaultValue));
```

Comment:
do not change existent **name**, but set translation for **displayName**
.set("displayName", getLocalization(definition?.get("displayName"), locale, defaultValue));